### PR TITLE
Add Kbox

### PR DESCRIPTION
**Pull request by hailin84**
Add Kbox desc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ Welcome to the Kunpeng compute community.
 | [hadoop](https://github.com/kunpengcompute/hadoop) | Apache Hadoop |  |
 | [kubernetes](https://github.com/kunpengcompute/kubernetes) | Production-grade container scheduling and management |  |
 | [arm-landscape](https://github.com/kunpengcompute/arm-landscape) | Track Arm CI landscape status for open source projects, including areas like big data, database, web, cloud, AI, etc. |  |
+| [Kbox](https://github.com/kunpengcompute/Kbox) | Android selinux patches for Kbox solution. | @Bessel-Native @ZhaoPengyuan-suffix @KnifeeOneOne |
```
Comment (Yikun) [inline README.md:45]: Please sign CLA and I will hold on it with -1 before our doc/i18n-ux team take a look.

**Review by Yikun — CHANGES_REQUESTED** [review]

**Review by rocmumu — CHANGES_REQUESTED** [review]
Android SELinux patches for the Kbox solution

**Comment by Yikun** [comment]
@hailin84 Please change the desciprtion according to @rocmumu suggestion.

And welcome to Kunpeng community.

File: README.md (hunk 1 above)
Comment (hailin84) [reply README.md:45]: CLA signed.

Comment (rocmumu) [inline README.md:45]: Android SELinux patches for the Kbox solution

Comment (rocmumu) [inline README.md]: Android SELinux patches for the Kbox solution

**Review by rocmumu — CHANGES_REQUESTED** [review]

**Pull request merged**